### PR TITLE
Add Nircmd,  do some useful tasks on the command line

### PR DESCRIPTION
**Pull request by pmiess**
add a nircmd feed.

nircmd is a poorly supported package with 16 packages across 5 repos

this is related to 0install/0install.de-feeds#3

### CODE REVIEW

**Comment by bastianeicher** [comment]
I would prefer to have the feed refer to the official download URLs, especially since they provide separate binaries for 32-bit and 64-bit:

http://www.nirsoft.net/utils/nircmd.zip
http://www.nirsoft.net/utils/nircmd-x64.zip

The downside is that the NirSoft website does not appear to offer older/versioned downloads. With a `.watch.py` file at least the feed would not be broken very long when a new version is released.

**Comment by pmiess** [comment]
That down side is why I chose this source. 
I'll work up a 0watch for the official downloads

**Comment by pmiess** [comment]
Added official feed, 0watch script and template.
Also removed some redundant elements. 
@bastianeicher Did you want me to remove the dependable sources entirely?

**Comment by bastianeicher** [comment]
I would suggest removing `<implementation arch="Windows-*" version="2.86" released="2019-08-01" id="alternative_source">` since Zero Install wouldn't have a clear way to determine whether it is "better" or "worse" than the official download.

Leaving the other implementations is fine by me.

**Comment by pmiess** [comment]
it is done

File: utils/nircmd.watch.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from urllib import request
+import re
+import datetime
+
+data = request.urlopen('http://www.nirsoft.net/utils/nircmd.html').read().decode('utf-8')
+matches = re.findall(r'<tr><td>(\d\d/\d\d/\d\d\d\d)<td>(\d.\d+)<td>', data)
+date = datetime.datetime.strptime(matches[0][0], '%d/%m/%Y').strftime('%Y/%m/%d')
```
Comment (bastianeicher) [inline utils/nircmd.watch.py:7]: This should be `.strftime('%Y-%m-%d')`.

File: utils/nircmd.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  <description xml:lang="en">By running NirCmd with simple command-line option, you can write and delete values and keys in the Registry, write values into INI file, dial to your internet account or connect to a VPN network, restart windows or shut down the computer, create shortcut to a file, change the created/modified date of a file, change your display settings, turn off your monitor, open the door of your CD-ROM drive, and more... </description>
+  <category>Utility</category>
+  <homepage>http://www.nirsoft.net/utils/nircmd.html</homepage>
+  <needs-terminal />
+  <group license="freeware">
+    <command name="run-gui" path="nircmd.exe"/>
+    <command name="run" path="nircmdc.exe"/>
+
+    <group version="2.86">
+      <implementation arch="Windows-i486" id="sha1new=5b65d612368b763a5987dd64c7de4af4d8eb4482" released="2019/08/01">
```
Comment (bastianeicher) [inline utils/nircmd.xml:14]: `released="2019-08-01"`

File: utils/nircmd.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        <manifest-digest sha256new="MY2TVRUOQ7JVNPAV7JURJQP63DWQHFRR4PACAIXJ2PJHE2HFMRVQ"/>
+        <archive href="http://www.nirsoft.net/utils/nircmd.zip" size="122984" type="application/zip"/>
+      </implementation>
+      <implementation arch="Windows-x86_64" id="sha1new=0c293bb7b168dd89e471e94248a622f82bf5934c" released="2019/08/01">
```
Comment (bastianeicher) [inline utils/nircmd.xml:18]: see above

File: utils/nircmd.watch.py (hunk 1 above)
Comment (pmiess) [reply utils/nircmd.watch.py]: fixed

File: utils/nircmd.xml (hunk 2 above)
Comment (pmiess) [reply utils/nircmd.xml]: fixed

File: utils/nircmd.xml (hunk 3 above)
Comment (pmiess) [reply utils/nircmd.xml]: fixed

**Comment by pmiess** [comment]
Your right I got the format wrong, sorry about that.
Fixed now.

**Pull request merged**